neo/Settings.py

```diff
@@ -78,15 +78,10 @@ def setup(self, config_file):
 
         self.config_file = config_file
 
-        try:
-            self.token_style = config['themes'][config['theme']]
-        except Exception as e:
-            self.token_style = {
-                "Command": "#ff0066",
-                "Default": "#00ee00",
-                "Neo": "#0000ee",
-                "Number": "#ffffff"
-            }
+        with open('preferences.json') as data_file:
```

Comment:
for reverse compatibility, could we add a check in the case that this file isn't located or is named something different?

Reply:
Just `if os.path.exists(fn):` would do the trick.

Reply:
I believe there is a little bit more to it than just checking if the file exists. If we're going to question if the file is there, then I think we should also question if the data inside is valid. Note that these checks also do not exist for non-existing protocol loading. Either way I'm looking into it.

Reply:
:+1:

Good idea to load the JSON and if it fails to give a warning or smth. If the keys are not found it should just use the default imo.

Since the existing protocol.json files don't change i guess they are safe as they are.

Reply:
the files itself might be safe yes, but `prompt` will throw up. Just try `python prompt.py -c nope`